src/pages/commits/ActiveCommits/CommitChart/index.tsx

```diff
@@ -1,8 +1,11 @@
 import styled from "@emotion/styled";
+import {
+  ColorCount,
+  calculateBarHeight,
+} from "pages/commits/ActiveCommits/utils";
```

Comment:
fwiw this `utils` file is probably a bit messed up in the first place since components outside of its parent folder call its functions (but that can be addressed at another time)

Reply:
Good catch i'll handle that in another ticket https://jira.mongodb.org/browse/EVG-20392